### PR TITLE
fs accounting: Add deleted files total size to status summary line - fixes issue #7190

### PR DESCRIPTION
**Pull request by KDreynolds**
#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: fs/accounting/stats.go [hunk 1]
```diff
@@ -464,7 +464,7 @@ func (s *StatsInfo) String() string {
 				s.checks, ts.totalChecks, percent(s.checks, ts.totalChecks))
 		}
 		if s.deletes != 0 || s.deletedDirs != 0 {
-			_, _ = fmt.Fprintf(buf, "Deleted:       %10d (files), %d (dirs)\n", s.deletes, s.deletedDirs)
+			_, _ = fmt.Fprintf(buf, "Deleted:       %10d (files), %d (dirs), %s freed\n", s.deletes, s.deletedDirs, fs.SizeSuffix(s.deletesSize).ByteUnit())
```
Comment (ncw) [inline fs/accounting/stats.go:467]: I think to be 100% consistent with the formating we'd put `freed` in brackends, so `(freed)`. However we could take them all out of brackets instead - what do you think?

**Review by ncw — COMMENTED** [review]
Nice change - thank you.

I put a question for discussion about the formatting (painting the bike-shed again) inline :-)

**Comment by KDreynolds** [comment]
ok I changed freed to be in parentheses. Should be good to go now I think. Lmk if you would like me to make any other changes here.

**Comment by ncw** [comment]
@KDreynolds it doesn't look like you've pushed that change?

**Review by ncw — APPROVED** [review]
Looks great, thank you :-)

**Pull request merged**